truss/truss_handle.py

```diff
@@ -863,7 +865,7 @@ def calc_patch(self, prev_truss_hash: str) -> Optional[PatchDetails]:
             return None
 
         prev_sign = TrussSignature.from_dict(json.loads(prev_sign_str))
-        patch_ops = calc_truss_patch(self._truss_dir, prev_sign)
+        patch_ops = calc_truss_patch(self._truss_dir, prev_sign, truss_ignore_patterns)
```

Comment:
It would be even better to calculate the ignored paths, and pass them into `calc_truss_patch`, `directory_content_hash`, and `calc_truss_signature`.